nixos/modules/services/misc/bees.nix

```
@@ -21,6 +21,8 @@ let
         <para>
         This must be in a format usable by findmnt; that could be a key=value
         pair, or a bare path to a mount point.
+        We prefer bare paths as then the beesd service will be ordered after
+        mounting the path itself.
```

This is a trivial matter, and the change has my :+1: regardless, but personally, I would word this as more like...
```suggestion
        Using bare paths will allow systemd to start the beesd service only
        after mounting the associated path.
```
...to avoid using the "royal we" in a way that could be read as speaking on the reader's behalf.